backends/mock.go

```diff
@@ -0,0 +1,31 @@
+package backends
+
+// mockBackend is used for tests
+type mockBackend struct {
+	Data []byte
+}
+
+func NewMockBackend() *mockBackend {
+	safeData := []byte(`-----BEGIN PGP MESSAGE-----
+wx4EBwMI/EyvqWA12cNgJBnoGRxYO1D0/F/w5Ro5uafS4AHkLjgl3wFVjIRB1vbo
```

Comment:
There should be an additional newline between `-----BEGIN PGP MESSAGE-----` and this line